app/views/jobResponsibility.html

```diff
@@ -0,0 +1,33 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% extends "layout_unbranded.html" %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+{% for resp in jobResponsibilty %}
```

Comment:
you don't need a look for this you can just say jobResponsibilty[0].name (this will change with my suggested API change)